oap-ape/ape-java/ape-common/src/main/java/com/intel/ape/service/netty/NettyMessage.java

```diff
@@ -677,7 +677,7 @@ private void writeCompressedDataToChannel(ChannelOutboundInvoker out,
                 compositeByteBuf.addComponent(nullBuffers[i]);
             }
             try {
-                ByteBuf compressedData = ZStdUtils.compress(compositeByteBuf);
+                ByteBuf compressedData = ICLCompressionUtils.compress(compositeByteBuf, "zstd");

```

Comment:
Is the second parameter of this API is codec type? if so, I would suggest use a pre-defined variable, rather than a hard-coded one.

Reply:
This should be one parameter passed from client request. We can add it in another PR.